src/shared/Core/Interop/Posix/GpgPassCredentialStore.cs

```diff
@@ -90,6 +111,86 @@ protected override void SerializeCredential(FileCredential credential)
 
             // Encrypt!
             _gpg.EncryptFile(credential.FullPath, gpgId, fileContents);
+
+            // If the store is a Git repository then add and commit the file addition or modification
+            if (TryGetGitRepository(out IGit git))
+            {
+                bool hasAccount = !string.IsNullOrWhiteSpace(credential.Account);
+
+                sb.Clear();
+                sb.AppendFormat("[GCM] Update credential {0}", credential.Service);
+                if (hasAccount)
+                {
+                    sb.AppendFormat(" ({0})", credential.Account);
+                }
+
+                sb.AppendLine();
+                sb.AppendLine();
+                sb.AppendFormat("Update Git Credential Manager credential:{0}", Environment.NewLine);
+                sb.AppendFormat("  Service: {1}{0}", Environment.NewLine, credential.Service);
+                if (hasAccount)
+                {
+                    sb.AppendFormat("  Account: {1}{0}", Environment.NewLine, credential.Account);
```

Comment:
Should these be reversed (as they are below)?
```suggestion
                    sb.AppendFormat("  Account: {1}{0}", credential.Account, Environment.NewLine);
```